### ISSUE TITLE
[BUG]: Winning Post 4 Maximum 2001, 5 Maximum 2002, presumably others: HW renders textures incorrectly

### ISSUE THREAD

**Issue by HeroponRikiBestest** [PCSX2/pcsx2#4923]
### Describe the Bug

In the two aforementioned Winning Post games, and likely others in the series, during a race, the hardware renderer renders the track incorrectly. I know the track texture looks more detailed on hardware, but from what I can tell, the software renderer is the correct one here. If you find evidence of otherwise, please let me know.

### Reproduction Steps

Wait on the title screen. A race demo will play after a few seconds.

### Expected Behavior

I expected the Hardware Renderer to display the track correctly.

### PCSX2 Revision

PCSX2 1.7.0-0 - compiled on Oct 18 2021 (daily ppa doesnt show git number)

### Operating System

Linux (64bit) - Specify Distro Below

### If Linux - Specify Distro

Linux Mint 20.2

### CPU

AMD Ryzen 7 PRO 4750U with Radeon Graphics

### GPU

integrated graphics, see above

### GS Window Screenshots

SW (4 2001):
![gs_20211021062929](https://user-images.githubusercontent.com/50224630/138263675-39cb7000-8a67-40bb-853f-c598d6f99303.png)
HW (4 2001):
![gs_20211021062946](https://user-images.githubusercontent.com/50224630/138263740-760f4c3f-4830-45da-886b-fc77f34c4a60.png)
SW (5 2002):
![gs_20211021062443](https://user-images.githubusercontent.com/50224630/138263862-298ebc02-8014-4c62-ba2b-d9000d6a51f1.png)
HW (5 2002):
![gs_20211021062647](https://user-images.githubusercontent.com/50224630/138263960-2a334547-38c4-4165-881d-8c188e9efcf0.png)

### Logs & Dumps

WP4 2001:
[gs_20211021062929.gs.zip](https://github.com/PCSX2/pcsx2/files/7388349/gs_20211021062929.gs.zip)

WP5 2002:
[gs_20211021062443.gs.zip](https://github.com/PCSX2/pcsx2/files/7388353/gs_20211021062443.gs.zip)

**Comment by HeroponRikiBestest** [comment]
Ok, setting mipmapping on Fast seems to fix it. I'll get the CRCs of all the affected games tomorrow, theres a lot to test.

**Comment by lightningterror** [comment]
Any update on this?

**Comment by lightningterror** [comment]
I went ahead and added the 2 games which were confirmed, for others a separate issue can be opened if they need it.
